contracts/libraries/Validator.sol

```diff
@@ -72,4 +31,83 @@ library Validator {
         }
         return computedHash == rootHash;
     }
+
+    // @param txHash      transaction hash
+    // @param rootHash    block header of the merkle tree
+    // @param input0      indicator for nonzero first input
+    // @param input1      indicator for nonzero second input
+    // @param sigs        transaction signatures
+    function checkSigs(bytes32 txHash, bytes32 confirmationHash, bool input0, bool input1, bytes sigs, bytes confirmSignatures)
+        internal
+        pure
+        returns (bool)
+    {
+        require(sigs.length == 130, "two transaction signatures required");
+        require(confirmSignatures.length == 130, "two confirm signatures required");
```

Comment:
Same as above for this. A one input tx results in one confirmation signature

Reply:
will do